qqqwww123cdv/hometask-3/rainfall.rb

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+def data
+  file = File.new('data.txt', 'r:UTF-8')
+  file.read
+end
+
+def weather_data(town, data)
+  data.split("\n").find do |value|
+    if value.include? town
+      monthly_rainfall = value.scan(/\d*\.\d/).map(&:to_f)
+      return monthly_rainfall
+    end
+  end
+end
+
+def mean(town, data)
+  return -1.0 unless data.match "#{town}:"
+
+  average_rainfall = weather_data(town, data)
```

Comment:
let's rename this variable since it's not `average` data and let's move it above the return to make us able to get rid of the redundant `data.match` operation